demos/tabs.html

```diff
@@ -0,0 +1,496 @@
+https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDC-Web Tabs Demo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="assets/material-components-web.css.js" charset="utf-8"></script>
+    <style>
+    	.demo-body {
+    	  padding: 24px;
+    	  margin: 0;
+    	  box-sizing: border-box;
+    	}
+
+      .demo-content > button {
+        margin-bottom: 6px;
+      }
+
+      .mdc-theme--dark {
+        background-color: #303030;
+      }
+
+      .mdc-theme--dark h1,
+      .mdc-theme--dark h2,
+      .mdc-theme--dark em,
+      .mdc-theme--dark .panel {
+        color: #a4a4a4;
+      }
+
+      .toggle-rtl {
+        margin-bottom: 32px;
+      }
+
+      #dynamic-demo-toolbar h1 {
+        margin-left: 64px;
+      }
+
+      #dynamic-demo-toolbar .mdc-tabs {
+        position: absolute;
+        bottom: 0;
+        margin-left: 20px;
+      }
+
+      section + section {
+        margin-top: 40px;
+      }
+
+      .panels {
+        padding: 8px;
+        border: 1px solid #ccc;
+        border-radius: 4px;
+        margin-top: 8px;
+      }
+
+      .panel {
+        display: none;
+      }
+
+      .panel.active {
+        display: block;
+      }
+
+      .dots {
+        display: flex;
+        justify-content: flex-start;
+        margin-top: 4px;
+        padding-bottom: 16px;
+      }
+
+      .dot {
+        margin: 0 4px;
+        border-radius: 50%;
+        border: 1px solid #ccc;
+        width: 20px;
+        height: 20px;
+      }
+
+      .dot:last-child {
+        margin-right: 0;
+      }
+
+      .dot.active {
+        background-color: blue;
+        border-color: blue;
+      }
+
+    </style>
+  </head>
+  <body class="mdc-typography">
+    <div class="demo-body">
+
+      <h1>Tabs</h1>
+      <button type="button" class="mdc-button mdc-button--primary" id="toggle-rtl">Toggle RTL</button>
+      <button type="button" class="mdc-button mdc-button--primary" id="toggle-dark-theme">Toggle Dark Theme</button>
+
+    <section>
+      <h2>Basic tabs</h2>
+      <nav id="basic-tabs" class="mdc-tabs">
+        <a class="mdc-tab mdc-tab--active" href="#one">Item One</a>
+        <a class="mdc-tab" href="#two">Item Two</a>
+        <a class="mdc-tab" href="#three">Three</a>
+        <span class="mdc-tabs__indicator"></span>
+      </nav>
+    </section>
+
+    <section>
+      <h2>CSS-Only Tabs</h2>
```

Comment:
Interesting. I can see that is a valid use case. 

Could we clearly document like above when you might suggest users using this css only method?